modules/sor/sorV2/sorPathService.ts

```diff
@@ -399,18 +425,19 @@ class SorPathService implements SwapService {
      * Fetch pools from Prisma and map to b-sdk BasePool.
      * @returns
      */
-    private async getBasePoolsFromDb(chain: Chain): Promise<PrismaPoolWithDynamic[]> {
+    private async getBasePoolsFromDb(chain: Chain, vaultVersion: number): Promise<PrismaPoolWithDynamic[]> {
         const poolIdsToExclude = AllNetworkConfigsKeyedOnChain[chain].data.sor?.poolIdsToExclude ?? [];
         const pools = await prisma.prismaPool.findMany({
             where: {
                 chain,
+                vaultVersion,
                 dynamicData: {
                     totalSharesNum: {
                         gt: 0.000000000001,
                     },
                     swapEnabled: true,
                     totalLiquidity: {
-                        gt: 1000,
+                        gte: vaultVersion === 2 ? 1000 : 0,
```

Comment:
should also do the same TVL limit for v3 pools

Reply:
Sorry, when I did that, all v3 pools had totalLiquidity<1000, I'll remove this now